archivessnake/unpublish_processing_note.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import os
+import json
+import time
+from asnake.aspace import ASpace
+
+
+aspace = ASpace(
```

Comment:
I'm a little unclear on how this is actually working? Or is the user supposed to add their configs in here?

Reply:
Oops - should be fixed now.